packages/boa/tools/install-python.js

```diff
@@ -27,10 +27,12 @@ if (!fs.existsSync(condaDownloadName)) {
   run(`curl ${CONDA_DOWNLOAD_PREFIX}/${condaDownloadName} > ${condaDownloadName}`);
 }
 
-run('rm', `-rf ${CONDA_LOCAL_PATH}`);
-run('sh', `./${condaDownloadName}`, `-f -b -p ${CONDA_LOCAL_PATH}`);
-run('rm', `-rf ${CONDA_LOCAL_PATH}/lib/libstdc++.so*`);
-run('rm', `-rf ${CONDA_LOCAL_PATH}/lib/libgcc_s.so*`);
+if (!(fs.existsSync(CONDA_LOCAL_PATH) && fs.existsSync(path.join(CONDA_LOCAL_PATH, 'bin', 'python')))) {
```

Comment:
Is 'fs.existsSync(CONDA_LOCAL_PATH)' unnecessary？

Reply:
My idea is if the CONDA_PATH and python bin in CONDA_PATH already exists, we do not need to delete it and install again to save time. Otherwise, remove the CONDA_PATH and download conda.

Reply:
Yeah, I agree with you and I think check 'fs.existsSync(path.join(CONDA_LOCAL_PATH, 'bin', 'python')))' is enough, e.g.:
```shell
if (!fs.existsSync(path.join(CONDA_LOCAL_PATH, 'bin', 'python'))) {
...
}
```